.travis.yml

```diff
@@ -30,6 +30,7 @@ install:
         sudo apt-get install $(./dist/tools/travis-scripts/get-pkg-list.py)
     - git config --global user.email "travis@example.com"
     - git config --global user.name "Travis CI"
+    - test "$TRAVIS_BRANCH" = "master" || git fetch origin $TRAVIS_BRANCH:$TRAVIS_BRANCH
```

Comment:
Without the check for master git will complain that it cannot fetch and
replace master while being on master. Due to the return value the build
would stop. Another solution would be to append "|| true"
Am 07.01.2016 19:53 schrieb "Joakim Nohlgård" <notifications@github.com>:

> In .travis.yml
> <https://github.com/RIOT-OS/RIOT/pull/4615#discussion_r49110156>:
>
> > @@ -30,6 +30,7 @@ install:
> >          sudo apt-get install $(./dist/tools/travis-scripts/get-pkg-list.py)
> >      - git config --global user.email "travis@example.com"
> >      - git config --global user.name "Travis CI"
> > +    - test "$TRAVIS_BRANCH" = "master" || git fetch origin $TRAVIS_BRANCH:$TRAVIS_BRANCH
>
> is the test necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RIOT-OS/RIOT/pull/4615/files#r49110156>.
>